common/coins_test.go

```diff
@@ -27,9 +27,12 @@ func TestEtherAmount(t *testing.T) {
 	require.Equal(t, amountUint, etherAmount.BigInt().Int64())
 }
 
-func TestToDecimals(t *testing.T) {
-	val := NewEtherAmount(123456)
-	require.Equal(t, "1.23456", FmtFloat(val.ToDecimals(5)))
-	val = NewEtherAmount(1234567890)
-	require.Equal(t, "1234.56789", FmtFloat(val.ToDecimals(6)))
+func TestERC20TokenAmount(t *testing.T) {
+	amount := float64(33.999999999)
```

Comment:
Can we add some more test values here like `33.000000005` and `33.0000000005`?

Reply:
yep, added more cases